rust/Cargo.toml

```diff
@@ -91,8 +91,12 @@ elliptic-curve = "0.12.3"
 getrandom = { version = "0.2", features = ["js"] }
 hex = "0.4.3"
 hpl-interface = "=0.0.6-rc3"
+# Must be the same version used by Tonic
+http = "0.2.9"
```

Comment:
I assume `http = "*"` wasn't working?

Reply:
i had a problem with it before but trying it again, even with a Cargo.lock from origin/main, works! nice